sandbox/registration-processor-mz.properties

```diff
@@ -377,11 +377,11 @@ registration.processor.reprocess.type=cron
 #schedular seconds configuration
 registration.processor.reprocess.seconds=0
 #schedular minutes configuration
-registration.processor.reprocess.minutes=0
+registration.processor.reprocess.minutes=0,5,10,15,20,25,30,35,40,45,50,55
 #schedular hours configuration
 # Under assumption of about 2 hrs of downtime for regproc on daily basis, the reprocessor 
 # frequency is made to 3 hours once
-registration.processor.reprocess.hours=0,3,6,9,12,15,18,21
```

Comment:
Change the above comments also, and add new comments why we are running it every 5 min

Reply:
I have changed the comment

Reply:
Merging as the Comment is changed along with reprocessor cron job timing.